test/e2e/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package e2e contains end-to-end tests for the cluster-kube-apiserver-operator.
+package e2e
```

Comment:
Why do we need this package?

Reply:
The doc.go file was added because:
  1. We had a blank import _ "github.com/openshift/cluster-kube-apiserver-operator/test/e2e" in main.go
  2. Go requires packages to have at least one non-test file when imported
  3. Without doc.go, the build fails with "no non-test Go files"